docs/inputs.rst

```diff
@@ -26,7 +26,7 @@ Orbit File
 -----------------
 
 .. note::
-  The orbit file is used by  **Objects in Field** and **Sorcha**.
+  The orbit file is used by **Sorcha**.
 
 This is a file which contains the orbital information of a set of synthetic objects. The orbital parameters must be **heliolcentric**, and orbits can be define in **Cometary(COM)  or Keplerian (KEP)** formats. Each simulated planetesimals within the synthetic population must be be given it's own unique object ID (ObjID). The file can be **white space separated**  or **comma value separated (CSV)** format. The first line of the orbit file is a header line starting with !! that specifies what each of the columns are.
```

Comment:
"each simulated **planetesimal**" and "must be given **its** own unique object ID"

Also, the first line of the orbit file should NOT start with !!.